.github/actions/azure-kubernetes-aks-single-region-cleanup/scripts/destroy-clusters.sh

```diff
@@ -77,7 +83,7 @@ destroy_cluster() {
     az graph query -q "
       ResourceContainers
       | where type == 'microsoft.resources/subscriptions/resourcegroups'
-      | where location =~ 'swedencentral'
+      | where location =~ '${AZURE_REGION}'
       | where createdTime < ago(${MIN_AGE_IN_HOURS}h)
```

Comment:
`createdTime` does not exist as key, so it would fail anyway as mentioned in the alert thread.

We also don't have to do this whole query thing for this particular cleanup.
We're already iterating through a known list of states and would just need to based on that delete the resource group.

It's all automated, so we know based on the `cluster_id` what the resource group name is.

E.g. if the `cluster_id` is `camunda-470-vanmo-aks` the the resource group is `camunda-470-vanmo-rg`.

Global cleanup, different topic but at least for this particular cleanup it would be sufficient.